website/docs/docs/dbt-versions/release-notes/72-Feb-2024/feb-sl-updates.md

```diff
@@ -0,0 +1,49 @@
+sidebar_label: "Updates and fixes: dbt Semantic Layer"
+sidebar_position: 05
+tags: [Feb-2024]
+date: 2024-02-29
+---
+
+The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and [MetricFlow](/docs/build/about-metricflow).
+
+The following list explains the new features, updates, and fixes for February 2024 in more detail.
+
+## dbt Semantic Layer
+**New features**
+- **Use exports to materialize saved queries** &mdash; [Exports](/docs/use-dbt-semantic-layer/exports#define-exports) allow you to materialize a saved query as a table or view in your data platform. By using exports, you can unify metric definitions in your data platform and query them as you would any other table or view.
+- **New export capabilities** &mdash; You can access a list of your [exports](/docs/use-dbt-semantic-layer/exports) with the new list saved-queries command by adding `--show-exports`
+- **Relative date filters in Tableau** &mdash; The dbt Semantic Layer and [Tableau Connector](/docs/use-dbt-semantic-layer/tableau) now supports relative date filters in Tableau.
+
+**Updates**
+
+- **Google Sheets integration** &mdash; The dbt Semantic Layer [Google Sheets integration](/docs/use-dbt-semantic-layer/gsheets) now exposes a note on the cell where the data was requested, indicating clearer data requests. The integration also now exposes a new **Time Range** option, which allows you to quickly select date ranges.
+- **GraphQL API enhancements** &mdash; The [GraphQL API](/docs/dbt-cloud-apis/sl-graphql) includes a `requiresMetricTime` parameter to better handle metrics that must be grouped by time. (Certain metrics defined in MetricFlow can't be looked at without a time dimension).
+
+**Bug fixes**
+
+- **Date filter with Tableau Connector** &mdash; The Tableau Connector returns a date filter when filtering by dates. Previously it was erroneously returning a timestamp filter.
+
+## MetricFlow
+
+**Updates**
+
+- **Metric querying** &mdash; Enable querying metrics with offset and cumulative metrics with the time dimension name, instead of `metric_time`. [Issue #1000](https://github.com/dbt-labs/metricflow/issues/1000)
+  - Enable querying `metric_time` without metrics. [Issue #928](https://github.com/dbt-labs/metricflow/issues/928)
+- **SQL generation** &mdash; Added support for consistent SQL query generation, which enables ID generation consistency between otherwise identical MF queries. Previously, the SQL generated by `MetricFlowEngine` was not completely consistent between identical queries. [Issue 1020](https://github.com/dbt-labs/metricflow/issues/1020)
+
+**Bug fixes**
+-  **Improved query parameter validation** &mdash; MetricFlow now validates if there are `metrics`, `group by`, or `saved_query` items in each query. Previously, there was no validation. [Issue 1002](https://github.com/dbt-labs/metricflow/issues/1002)
```

Comment:
This feels like an update and not a bug?

Reply:
yea good flag! i feel like it could be both?! the mf pr labels it as a bug fix so I'll keep it as that for consistency if that's ok! i guess its how i phrased it. the way i see it, since there wasn't validation initially and caused errors/unexpected behavior, then fixing it is fixing the bug?